infra/stage/locals.tf

```diff
@@ -1,7 +1,10 @@
 locals {
-  project_name           = "gpo"
-  environment            = "stage"
+  project_name = "gpo"
+  environment  = "stage"
+  /* eks specific */
   nodegroup_desired_size = 1
   nodegroup_max_size     = 5
   nodegroup_min_size     = 1
+  /* ecr specific */
+  repositories = ["donor-history-report"]
```

Comment:
Is this the name of the container? The `gpo/gpo` repo will do a lot more than just the current report. I wonder if we should call it `gpo-monolith`?

Reply:
It is the name of the container. Do we expect to build a single container for every project that lives in the repo? I feel like it's more common to have separate Dockerfiles for each project in a monorepo.

Reply:
It is a major drag about ECR though that you have to add a new repository manually every time you want to push a brand new container name :/

Reply:
NestJS is more of a monolith style framework so it'll be a single image for most of the repo.

There might be a couple other docker images that come out of this repo, like one for 2e2 tests, but that's the only other one I can think of.

Reply:
Ahh so we don't plan to have like a PHP script or a python app in here too? Yeah in that case `gpo-monolith` makes sense.

Reply:
Yeah, probably not. I looked into the mono-repo pattern and it seems like the overhead would be higher than the value if we keep the number of repos small.